README.md

```diff
@@ -35,27 +35,28 @@ Development version:
 remotes::install_github("Appsilon/rhino")
 ```
 
-## Usage
+## Documentation
 
-* Create a new Rhino application with `rhino::init()`
-* To learn more, follow the
-[Rhino tutorial](https://appsilon.github.io/rhino/articles/tutorial/create-your-first-rhino-app.html)
-* To migrate an existing application to Rhino, refer to
-[`rhino::init()` details section](https://appsilon.github.io/rhino/reference/init.html#details-1)
+[Rhino documentation](https://appsilon.github.io/rhino/) is divided into four sections,
+accessible from the navigation bar at the top:
 
-## About
-
-Rhino is distributed under [LGPL-3.0 license]. Developed with :heart: at [Appsilon].
+1. [**Tutorial**](https://appsilon.github.io/rhino/articles/tutorial/create-your-first-rhino-app.html):
+Start here to learn the basics.
+2. **Explanation**:
```

Comment:
It would be great if all section titles here were clickable (e.g. go to the top of the page and open the section dropdown).

Reply:
I thought about that too :wink: It's challenging though, as this `README` is also displayed on GitHub. Would require some JS magic probably. I'll skip on this for now.